MiniTeks/mks-server
# Add controller for MksPipelineRun

### Add controller for MksPipelineRun

- Listens for MksPipelineRun resource being created, updated, or deleted.
- Adds corresponding TaskPipeline using Tekton API.

### How to test

- Build and run the executable using steps in README.md.
- Apply the CRDs and examples in the config folder.
- Add a new MksPipelineRun resource and see for logs.

### Related issues

Fixes #12
